creme/stream/modif-bouteille/sql.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy import create_engine, MetaData, select
+import os
+
+def iter_sql(con):
+    """Yields rows from all tables in a database.
+    
+    Parameters:
+        con: SQLAlchemy connectable (engine/connection) or database str URI.
```

Comment:
You should specify the type in the docsting :) 

```python
Parameters:
   param1 (type): description

```

Reply:
The type of `con` should be [`sqlalchemy.engine.Engine`](https://docs.sqlalchemy.org/en/13/core/connections.html#sqlalchemy.engine.Engine).